vimeo-networking/src/main/java/com/vimeo/networking/Search.java

```diff
@@ -205,16 +202,27 @@ public static FilterCategory fromString(String text) {
     static final String FILTER_DURATION = "filter_duration";
     static final String FILTER_TYPE = "filter_type";
     static final String FILTER_FEATURED_COUNT = "featured_clip_count";
+    static final String PARAMETER_GET_FACETS = "facets";
 
     public static Call<SearchResponse> search(@NotNull String query, @NotNull FilterType type,
                                               @NotNull ModelCallback<SearchResponse> callback,
                                               @Nullable Map<String, String> refinementMap,
-                                              @Nullable String containerFilter,
+                                              @Nullable List<Facet> facets, @Nullable String containerFilter,
                                               @Nullable String fieldFilter) {
         if (refinementMap == null) {
             refinementMap = new HashMap<>();
         }
         refinementMap.put(FILTER_TYPE, type.getText());
+        if (facets != null) {
+            String facetsToUse = "";
+            for (int i = 0; i < facets.size(); i++) {
+                facetsToUse += facets.get(i).getText();
+                if (i + 1 != facets.size()) {
+                    facetsToUse += ",";
+                }
+            }
```

Comment:
👍 Nice, I'll go with this, it's clean. I also considered using StringUtils (its a sun package) that can do the same, but I opted to not include any extra imports and write this sucker by hand.